src/ui.js

```diff
@@ -46,7 +47,8 @@ export default class Ui {
 
   /**
    * CSS classes
-   * @constructor
+   *
+   * @class
```

Comment:
@class annotation is intended to be called with the "new" keyword, it is inappropriate here